content/en/docs/setup/independent/install-kubeadm.md

```diff
@@ -165,7 +165,7 @@ For more information on version skews, please read our
 apt-get update && apt-get install -y apt-transport-https curl
 curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
 cat <<EOF >/etc/apt/sources.list.d/kubernetes.list
-deb http://apt.kubernetes.io/ kubernetes-xenial main
+deb https://packages.cloud.google.com/apt/ kubernetes-xenial main
```

Comment:
It smells to me that the *.kubernetes.io domain names are there for the ease of remembering. Not sure if https://apt.kubernetes.io/ can be used instead?